src/app/core/config/config-fix.ts

```diff
@@ -1,4 +1,8 @@
-{
+import { defaultAttendanceStatusTypes } from "./default-config/default-attendance-status-types";
+import { defaultInteractionTypes } from "./default-config/default-interaction-types";
+
+// prettier-ignore
+export const defaultConfig = {
```

Comment:
we should be careful to never put any logic in this file

Reply:
that's an important remark. After you raised this I looked at typescript typings for it, but it seems to be far from trivial to define a JSON type ...